composer.json

```diff
@@ -40,5 +40,10 @@
                 "JeroenG\\Autowire\\AutowireServiceProvider"
             ]
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "infection/extension-installer": true
```

Comment:
Could you disable this?

Reply:
Sure, but without it, the pipelines were all red, as composer failed to run.

See here: https://github.com/Jeroen-G/autowire/actions/runs/3731905388/jobs/6330660232

Reply:
Or do you mean swapping it for
```
"infection/extension-installer": false,
```
?

Reply:
In case of the latter, shall I add `--no-plugins` to the composer calls in `cy.yml` ?

Reply:
Yes and OK!

Reply:
Done